virttest/qemu_devices/qcontainer.py

```diff
@@ -1785,6 +1804,9 @@ def pcic_by_params(self, name, params):
             elif driver == 'i82801b11-bridge':  # addr 0x1-0x13
                 bus_length = 20
                 bus_first_port = 1
+            elif driver in ('pcie-root-port', 'ioh3420'):
+                bus_length = 1  # multifunction off by default
```

Comment:
How to turn on the multifunction?

Reply:
@YongxueHong It is only implemented here: https://github.com/avocado-framework/avocado-vt/blob/master/virttest/qemu_devices/qdevices.py#L1854 Not fully supported now in pcie bus level, you might open a bz and assign it to me if you have such requirement. Thanks.

Reply:
@vivianQizhu 
OK, thanks.